### ISSUE TITLE
False positive/unexpected indent error

### ISSUE THREAD

**Issue by paulschreiber** [WordPress-Coding-Standards/WordPress-Coding-Standards#291]
I'm getting some unexpected indent errors on lines 38 and 39 here:
```--------------------------------------------------------------------------------
FOUND 3 ERRORS AND 2 WARNINGS AFFECTING 5 LINES
--------------------------------------------------------------------------------
 31 | ERROR   | [ ] Detected high pagination limit, `posts_per_page` is set to
    |         |     `100` (WordPress.VIP.PostsPerPage.posts_per_page)
 32 | WARNING | [ ] Detected usage of meta_key, possible slow query.
    |         |     (WordPress.VIP.SlowDBQuery.slow_db_query)
 33 | WARNING | [ ] Detected usage of meta_value, possible slow query.
    |         |     (WordPress.VIP.SlowDBQuery.slow_db_query)
 38 | ERROR   | [x] Line indented incorrectly; expected 0 tabs, found 4
    |         |     (Generic.WhiteSpace.ScopeIndent.IncorrectExact)
 39 | ERROR   | [x] Line indented incorrectly; expected 1 tabs, found 5
    |         |     (Generic.WhiteSpace.ScopeIndent.IncorrectExact)
--------------------------------------------------------------------------------
PHPCBF CAN FIX THE 2 MARKED SNIFF VIOLATIONS AUTOMATICALLY
--------------------------------------------------------------------------------
```

Here's the code:

```php
 23             <?php
 24                 the_content();
 25                 endwhile;
 26 
 27                 global $coauthors_plus;
 28 
 29                 $author_query = new WP_Query(array(
 30                         'post_type' => 'guest-author',
 31                         'posts_per_page' => 100,
 32                         'meta_key' => $coauthors_plus->guest_authors->get_post_meta_key( 'contributors_page' ),
 33                         'meta_value' => 1,
 34                 ));
 35 
 36                 $authors = array();
 37 
 38                 if ( $author_query->have_posts() ) {
 39                     while ( $author_query->have_posts() ) {
 40                         $author_query->the_post();
 41                         $authors[] = $coauthors_plus->get_coauthor_by( 'ID', get_the_ID() );
 42                     }
 43                 }
```

If I close and open the php block like so:

```php
 23             <?php
 24                 the_content();
 25                 endwhile;
 26                 ?>
 27 
 28                 <?php // close and open this tag to avoid PHPCS warning
 29                 global $coauthors_plus;
```

The indent errors go away.

**Comment by westonruter** [comment]
@paulschreiber what are lines 1–22 in your sample file?

**Comment by paulschreiber** [comment]
Here are lines 1-23:

```php
<?php
/**
 * Contributors Template
 * Template Name: Contributors
 *
 * @package FiveThirtyEight
 */
get_header();
get_template_part( 'template-parts/masthead' );
?>
<div id="content" class="home" data-col="2">
    <div id="primary" class="single-col">
        <div id="main" class="content-area">
            <div class="page-header">
                <h1 class="page-title">
                    <?php while ( have_posts() ) : the_post();
                        echo the_title();
                    ?>
                </h1>
            </div>

            <div class="entry-content">
            <?php
```

**Comment by westonruter** [comment]
I'm not 100% sure, but to me it seems the template PHP logic in the template doesn't seem organized properly. Isn't it better to organize the `while` loop like this?

```php
<?php
/**
 * Contributors Template
 * Template Name: Contributors
 *
 * @package FiveThirtyEight
 */
get_header();
get_template_part( 'template-parts/masthead' );
?>
<div id="content" class="home" data-col="2">
	<div id="primary" class="single-col">
		<div id="main" class="content-area">
			<?php while ( have_posts() ) : the_post(); ?>
				<div class="page-header">
					<h1 class="page-title">
						<?php the_title(); ?>
					</h1>
				</div>

				<div class="entry-content">
					<?php the_content(); ?>
				</div>
			<?php endwhile; ?>
			
			<?php
			global $coauthors_plus;

			$author_query = new WP_Query(array(
					'post_type' => 'guest-author',
					'posts_per_page' => 100,
					'meta_key' => $coauthors_plus->guest_authors->get_post_meta_key( 'contributors_page' ),
					'meta_value' => 1,
			));

			$authors = array();

			if ( $author_query->have_posts() ) {
				while ( $author_query->have_posts() ) {
					$author_query->the_post();
					$authors[] = $coauthors_plus->get_coauthor_by( 'ID', get_the_ID() );
				}
			}
```

**Comment by JDGrimes** [comment]
I'm not able to reproduce. Updating PHPCS might help (see #364). `Generic.WhiteSpace.ScopeIndent` is part of PHPCS, and not our sniffs, so unless we're configuring it wrong, the problem (if it still exists) is probably upstream.

**Comment by paulschreiber** [comment]
PHPCS confirmed it's their bug (and fixed it):
https://github.com/squizlabs/PHP_CodeSniffer/issues/515
